src/main/java/io/cdap/plugin/googleads/source/batch/GoogleAdsBatchSourceConfig.java

```diff
@@ -155,36 +158,70 @@ protected void validateAuthorisation(FailureCollector failureCollector, GoogleAd
   }
 
   protected void validateReportTypeAndFields(FailureCollector failureCollector, GoogleAdsHelper googleAdsHelper) {
-    if (containsMacro(REPORT_TYPE)){
+    if (containsMacro(REPORT_TYPE)) {
       return;
     }
     ReportDefinitionReportType reportDefinitionReportType = null;
     try {
       reportDefinitionReportType = getReportType();
     } catch (IllegalArgumentException ex) {
-      failureCollector.addFailure("Invalid reportDefinitionReportType", "Enter valid reportDefinitionReportType").withConfigProperty(REPORT_TYPE);
+      failureCollector.addFailure("Invalid reportDefinitionReportType",
+                                  "Enter valid reportDefinitionReportType")
```

Comment:
Corrective action does not provide any new action. Hence is redundant and should be null.